### PR TITLE
Full CICD Pipeline #19

### PR DESCRIPTION
**Pull request by eoinsha**

### CODE REVIEW

File: cicd/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# This file is for unifying the coding style for different editors and IDEs
```
Comment (pelger) [inline cicd/.editorconfig]: Why are we including this? Should be removed?

File: cicd/config.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const accountIds: { [stageName: string]: number } = {
```
Comment (pelger) [inline cicd/config.ts]: Seems like a bad idea to hard code account IDs into this file. Should they not be included as environment settings?

File: cicd/config.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  cicd: 285982925560,
+  dev: 935672627075,
+  stg: 835483165098,
+  prod: 302391791660
+}
+
+export default {
+  stage: process.env.SLIC_STAGE || 'dev',
```
Comment (pelger) [inline cicd/config.ts]: Same here - should these not be in the environment?

File: cicd/cross-account/serverless.yml [hunk 4]
```diff
@@ -1,13 +1,18 @@
 service: deployment-target-resources
 
+custom:
+  stage: ${opt:stage, 'cicd'}
+  region: eu-west-1
+  runtime: nodejs8.10
+  accountIds:
```
Comment (pelger) [inline cicd/cross-account/serverless.yml]: Environment also for this account id?

File: cicd/config.ts (hunk 2 above)
Comment (eoinsha) [reply cicd/config.ts]: We can parameterise them with environment variables, config files, etc?
Curious why it's a bad idea? For a given organisation, these account IDs will be static. Would there be a reason to dynamically change the account for production, for example and not have that change-managed in version control?

**Pull request merged**